frontend/src/views/GShootItemTerminal.vue

```diff
@@ -24,6 +32,13 @@ export default {
       ...this.terminalSplitpanes,
     }
   },
+  beforeRouteEnter (to, from, next) {
+    next(vm => {
+      if (!vm.isAdmin && !vm.hasShootWorkerGroups) {
+        vm.$router.replace({ name: 'ShootItem', params: vm.$route.params })
```

Comment:
this results in a strange navigation endless loop

Reply:
I discovered this, too but I don't know why. We have similar coding in `GLogin` component.